secrets-provider-test-chart/values.yaml

```diff
@@ -0,0 +1,25 @@
+global:
+  namespace: local-secrets-provider
```

Comment:
values should be valid defaults for customers to use.
I would remove the `local-` prefix.